include/loops/loops.hpp

```diff
@@ -435,6 +535,14 @@ static inline IReg& operator ^= (IReg& a, const IReg& b)
 
 // [TODO] need to add type conversion (including expansion etc.), type reinterpretation
 
+inline uint64_t makeBitmask64(std::initializer_list<size_t> regNumbers)
```

Comment:
why do we need it in the external header?

Reply:
Deleted.